doc/API.md

```diff
@@ -21,6 +21,11 @@
 	1. [API Name: axe.registerPlugin](#api-name-axeregisterplugin)
 	1. [API Name: axe.cleanup](#api-name-axecleanup)
 	1. [API Name: axe.a11yCheck](#api-name-axea11ycheck)
+	1. [Virtual DOM Utilities](#virtual-dom-utilities)
+		1. [API Name: axe.utils.getFlattenedTree](#api-name-axeutilsgetflattenedtree)
+		2. [API Name: axe.utils.getNodeFromTree](#api-name-axeutilsgetnodefromtree)
+		3. [API Name: axe.utils.querySelectorAll](#api-name-axeutilsqueryselectorall)
+		4. [API Name: axe._tree](#api-name-axe_tree)
```

Comment:
I might just remove it for now so we don't hold up the release. Because I have questions about how that would work, since it gets set internally during the audit: 1. when would you set or clear it? 2. what should happen if you try to manipulate it at the wrong time?